### PR TITLE
Port fix of flake8 to branch master

### PR DESCRIPTION
**Pull request by nbessi**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+language: python
+python:
+  - "2.7"
+
+virtualenv:
+  system_site_packages: true
+
+before_install:
+ - git clone https://github.com/nbessi/maintainer-quality-tools.git $HOME/maintainer-quality-tools -b consolidated
+ - git clone https://github.com/OCA/connector $HOME/connector -b 7.0
```
Comment (bwrsandman) [inline .travis.yml:10]: Are you sure you want 7.0 here?

Comment (bwrsandman) [inline .travis.yml:10]: Adding to that: this is the master (8.0 ?) branch, so the `-b 7.0` probably needs to be fixed

**Comment by nbessi** [comment]
@bwrsandman is correct I targeted wrong branch

**Comment by nbessi** [comment]
The travis modification must be reverted

**Comment by gurneyalex** [comment]
I've created 8.0 and removed master. Please resubmit on 8.0

**Pull request closed without merging**